base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
> My concern now is that the method I'm documenting shows up as `prod(f::Callable, itr)`, though `Callable` is not exported.

In the URL for the particular docstring, yes it will list `Callable` as part of it's signature, but that's just required to uniquely identify the docstring. In the actual text of the docstring you don't need to include the word `Callable` since the handwritten signatures aren't used by the docsystem any more like they were in `genstdlib.jl`. Could just write

> ... where `f` is callable.

perhaps as part of the description instead.

Regarding the missing `sum` docstring, probably due to Sphinx not producing an URL for that specific signature in the docs, which resulted in the translation not picking it up. I'm sure there's probably a couple of those that slipped through, so best thing to do there is just add them back whenever someone notices one.